test/output/name/expected.js

```diff
@@ -1,4 +1,4 @@
-"use strict";var module1=module;module1.export({foo:()=>foo});module1.export({id:()=>id,name:()=>name},true);const path = require("path");
+"use strict";module.export({foo:()=>foo});module.export({id:()=>id,name:()=>name},true);const path = require("path");
```

Comment:
As you can see from these updated tests, many of the `const module1 = module` renamings were unnecessary false positives before this change.